src/Bicep.Cli/Constants/CliConstants.cs

```diff
@@ -11,6 +11,7 @@ public static class Command
         public const string Format = "format";
         public const string JsonRpc = "jsonrpc";
         public const string LocalDeploy = "local-deploy";
+        public const string Deploy = "deploy";
```

Comment:
Should we set up `validate` & `what-if` at the same time?

Reply:
Added